src/App.vue

```diff
@@ -1,9 +1,13 @@
+<script setup lang="ts">
+import CitySearch from "@/components/city-search/CitySearch.vue";
+import BreadcrumbsBar from "@/components/breadcrumbs-bar/BreadcrumbsBar.vue";
+</script>
+
 <template>
-    <nav>
-        <router-link to="/">Home</router-link> |
-        <router-link to="/about">About</router-link> |
-        <router-link to="/dashboard">Dashboard</router-link>
-    </nav>
+    <div class="city-search-wrapper">
+        <CitySearch />
```

Comment:
There could be a bit more space between the search bar and the heading in general

<img width="443" alt="image" src="https://github.com/sickinga/hux_weather/assets/48798335/295960d3-2c42-4c8a-bff5-95501e3bffe9">